packages/core/src/events.ts

```diff
@@ -35,7 +37,7 @@ export const completeEvent = (event: Event, store: Store<any, any>): void => {
 
   store.universe = nextUniverse;
   event.nextUniverse = nextUniverse;
-  submitPatches(store, store.universe, event.patches);
+  submitPatches(store, store.universe, event.patches, event);
```

Comment:
No need to pass in both `event.patches` and `event` here. Can just pass in `event`